web-server/src/slices/service.ts

```diff
@@ -0,0 +1,76 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { ServiceNames } from '@/constants/service';
+
+type Status = {
+  isUp: boolean;
+};
+
+type Service = {
+  isUp: boolean;
+  logs: string[];
+};
+
+export type ServiceStatusState = {
+  [key in ServiceNames]: Service;
+};
```

Comment:
optional:
can be easily written as Record<ServiceNames, Service>

Reply:
Sure will do that